src/machines/modelingMachine.ts

```diff
@@ -531,16 +532,16 @@ export const modelingMachine = setup({
         }
       }
     ),
-    // eslint-disable-next-line @typescript-eslint/no-misused-promises
-    'hide default planes': () => kclManager.hidePlanes(),
+    'hide default planes': () => {
+      void kclManager.hidePlanes()
+    },
     'reset sketch metadata': assign({
       sketchDetails: null,
       sketchEnginePathId: '',
       sketchPlaneId: '',
     }),
     'reset camera position': () => {
-      // eslint-disable-next-line @typescript-eslint/no-floating-promises
-      engineCommandManager.sendSceneCommand({
+      void engineCommandManager.sendSceneCommand({
```

Comment:
Same here, we should leave the ignore comments to make it easier to find where we need to move actions to actors